types/googlemaps/reference/event.d.ts

```diff
@@ -119,48 +119,143 @@ declare namespace google.maps {
         unbindAll(): void;
     }
 
-    /** This class extends MVCObject. */
+    interface MVCArrayHandlerMap<C extends MVCArray<T>, T> {
+        /**
+         * This event is fired when {@link MVCArray.insertAt insertAt}() is called. The event passes the index that was
+         * passed to {@link MVCArray.insertAt insertAt}().
+         * @see {@link https://developers.google.com/maps/documentation/javascript/reference/event#MVCArray.insert_at Maps JavaScript API}
+         */
+        insert_at: (this: C, index: number) => void;
+
+        /**
+         * This event is fired when {@link MVCArray.removeAt removeAt}() is called. The event passes the index that was
+         * passed to {@link MVCArray.removeAt removeAt}() and the element that was removed from the array.
+         * @see {@link https://developers.google.com/maps/documentation/javascript/reference/event#MVCArray.remove_at Maps JavaScript API}
+         */
+        remove_at: (this: C, index: number, removed: T) => void;
+
+        /**
+         * This event is fired when {@link MVCArray.setAt setAt}() is called. The event passes the index that was passed
+         * to {@link MVCArray.setAt setAt}() and the element that was previously in the array at that index.
+         * @see {@link https://developers.google.com/maps/documentation/javascript/reference/event#MVCArray.set_at Maps JavaScript API}
+         */
+        set_at: (this: C, index: number, previous: T) => void;
+    }
+
+    /**
+     * @see {@link https://developers.google.com/maps/documentation/javascript/reference/event#MVCArray Maps JavaScript API}
+     */
     class MVCArray<T> extends MVCObject {
```

Comment:
@jpoehnelt. The prototype of the `MVCArray` object contains the `indexOf` method but is not included in [the documentation](https://developers.google.com/maps/documentation/javascript/reference/event#MVCArray). Is this method deprecated?

<img width="453" alt="Console screen" src="https://user-images.githubusercontent.com/10235949/95227616-fe0ca680-0806-11eb-8855-fab6c9290b74.png">